kickstarter/android-oss
# NTV-298:Layout breaking after entering pledge flow

# 📲 What
Fix weird gap on the bottom of the screen

# 🛠 How
Calculate animation 

# 👀 See

https://user-images.githubusercontent.com/1075310/143285382-bbde8c93-500d-49bc-b6e7-413e6c58b453.mp4


# 📋 QA

1. Open a project on the new Project screen
2.  Tap Back this project 
3. Notice there's a weird gap on the bottom of the screen


# Story 📖
https://kickstarter.atlassian.net/browse/NTV-298
